main.js

```diff
@@ -160,4 +163,115 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+//opdracht 1a
+console.log("opdracht 1a");
+const output = inventory.map(tv => tv.name);
+console.log(output);
+
+
+//opdracht 1b
+console.log("opdracht 1b");
+inventory.map((tv) => {
```

Comment:
Als je niets met de uitkomst van de map functie doet, kun je beter een forEach gebruiken
```suggestion
inventory.forEach((tv) => {
```